src/openrct2/world/TileInspector.cpp

```diff
@@ -359,7 +359,7 @@ namespace OpenRCT2::TileInspector
     GameActionResultPtr PasteElementAt(const CoordsXY& loc, TileElement element, bool isExecuting)
     {
         // Make sure there is enough space for the new element
-        if (!!MapCheckCapacityAndReorganise(loc))
```

Comment:
I am surprised none of the compilers give a warning for this.

Reply:
Interesting indeed 🤔